src/frontend/next/src/components/SearchPage/SearchPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC, FormEvent, useState } from 'react';
+import { useQueryParam, StringParam } from 'use-query-params';
+import { makeStyles } from '@material-ui/core/styles';
+
+import SearchResults from './SearchResults';
+import SearchBar from '../SearchBar';
+import BackToTopButton from '../BackToTopButton';
+
+const useStyles = makeStyles(() => ({
+  anchor: {
+    position: 'absolute',
+    top: 0,
+  },
+}));
+
+const SearchPage: FC = () => {
+  const classes = useStyles();
+  // We synchronize the `text` and `filter` values to the URL's query string
+  // via `textParam` and `filterParam`. The <SearchBar> UI uses our internal
+  // state values, and the <SearchResults> only update on page load or when
+  // the user submits the form.
+  const [textParam = '', setTextParam] = useQueryParam('text', StringParam);
```

Comment:
So I merged in the current master to this pr and I'm getting errors when trying to render ` <SearchPage />`:

`Error: useQueryParams must be used within a QueryParamProvider`

Got any ideas?